specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -1180,6 +1229,129 @@
       "additionalProperties": true,
       "description": "Captions are the most representative passages from the document relatively to the search query. They are often used as document summary. Captions are only returned for queries of type 'semantic'.."
     },
+    "AnswersDebugInfo": {
+      "properties": {
+        "queryClassifiedAsQuestion": {
+          "type": "boolean",
+          "description": "Whether the given query was classified as a question.",
+          "readOnly": true
+        }
+      },
+      "x-ms-client-name": "Answers",
+      "description": "Answers debug info for this search result."
+    },
+    "SemanticDebugInfo": {
+      "x-ms-client-name": "Semantic",
+      "description": "Semantic debug info for this search result.",
+      "$ref": "#/definitions/AnswersDebugInfo"
+    },
+    "RerankerInputResult": {
+      "properties": {
+        "title": {
+          "type": "string",
+          "readOnly": "true",
+          "description": "The title field."
+        },
+        "content": {
+          "type": "string",
+          "readOnly": "true",
+          "description": "The concatenated content fields."
+        },
+        "keywords": {
+          "type": "string",
+          "readOnly": "true",
+          "description": "The concatenated keyword fields."
+        }
+      },
+      "description": "The concatenated semantic fields sent to the reranker model.",
+      "x-ms-client-name": "RerankerInput"
+    },
+    "SemanticFieldResult": {
+      "x-ms-client-name": "Semantic",
+      "properties": {
```

Comment:
I tried to generate the .NET SDK using this swagger and I'm getting below error-
`error   | PreCheck/DuplicateSchema | Duplicate Schema named 'Semantic' (6 differences)`

![image](https://user-images.githubusercontent.com/45216704/205157765-0dca7ce4-a990-42f4-845a-f705d38d6364.png)

Reply:
Thanks for letting me know! Are you following these steps to generate the SDK, or are you using another toolchain? https://github.com/Azure/azure-rest-api-specs/tree/master/specification/search/data-plane/Azure.Search

I'd like to be able to generate a copy of the SDK and validate things locally to speed up my dev workflow for swagger.

Reply:
You can run `dotnet build /t:GenerateCode` in the [src](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/search/Azure.Search.Documents/src) directory as mentioned [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/src/autorest.md) to generate .NET SDK code.

[Here](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#prerequisites) are the prerequisites to generate code and work on the `azure-sdk-for-net` repo.